CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 # intellij-stepbuilder-codegen-plugin Changelog
 
+### [1.1.6]
```

Comment:
Regarding the release workflow this tag has to be: `[Unreleased]` (Then the release plugin will automatically increase the version)